components/library/libmng/libmng.p5m

```diff
@@ -12,6 +12,7 @@
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
 # Copyright 2018 Harry Liebel <hliebel@gmail.com>
+# Copyright 2019 Tim Mooney <Timothy.V.Mooney@gmail.com>
```

Comment:
You didn't actually change here anything but the copyright :).

Reply:
Sorry, I couldn't find it spelled out in the documentation whether I needed to update copyright in each of the files, or just any file that had it that I touched (like Makefile).

I'll submit a new pull request with those two copyright diffs removed.

Reply:
> Sorry, I couldn't find it spelled out in the documentation whether I needed to update copyright in each of the files, or just any file that had it that I touched (like Makefile).

Yes, it's the latter. CDDL is file-based license.

> I'll submit a new pull request with those two copyright diffs removed.

Please reuse this PR, for consistency reasons.